x-pack/solutions/security/plugins/security_solution/public/timelines/components/timeline/unified_components/data_table/index.tsx

```diff
@@ -237,9 +236,8 @@ export const TimelineDataTableComponent: React.FC<DataTableProps> = memo(
     );
 
     const handleFetchMoreRecords = useCallback(() => {
-      onFetchMoreRecords(fetchedPage + 1);
```

Comment:
does it need to be wrapped with useCallback at this point?

Reply:
Why not? `handleFetchMoreRecords` is passed to the table component and depends on `onFetchMoreRecords`. So i this it should still be enclosed in `useCallback`.

why cause an extra re-render? Let me know if I missed something.